File: bitSetGo/D-CropCirclesAndTryangles/logic.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {

    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
         Scanner in=new Scanner(System.in);
         int t=in.nextInt();
         while(t>0)
         {
             int r=in.nextInt();
             double ar=0.4194*r*r;
             System.out.printf("%.2f\n",ar);
         }
    }
}

File: bitSetGo-v2.0/B-Tricky-Proposal/logic.py
n = 1;
t = int(input());

for i in range(t):
    x = int(input());
    
    while x >= ((n+1)*(n+2)/2):
        print(n,end=" ")
        n = n+1

    print(int(x-(n*(n-1))/2))
    n = 1;
